spec/system/creator_spec.rb

```diff
@@ -74,6 +75,9 @@
     end
     it "can edit an existing creator" do
       creator
+      creator_with_alternates
```

Comment:
I'm confused why this change is needed? Were we going to add another test here thad didn't get finished?